pkgs/servers/jibri/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchurl, dpkg, jre_headless }:
+
+let
+  pname = "jibri";
+  version = "8.0-93-g51fe7a2";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
+    sha256 = "1w78aa3rfdc4frb68ymykrbazxqrcv8mcdayqmcb72q1aa854c7j";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  dontBuild = true;
+
+  unpackCmd = "${dpkg}/bin/dpkg-deb -x $src debcontents";
```

```suggestion
  unpackCmd = "dpkg-deb -x $src debcontents";
```
and add dpkg to nativeBuildInputs.